Onboarding/profile-pregnancy-health.md

```diff
@@ -33,23 +34,27 @@ version: "0.1.0"
 columns: [] 
 -->
 
-| Key               | Value                                   |
-| ----------------- |-----------------------------------------|
-| contentrepo_token | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+| Key               | Value                                    |
+| ----------------- | ---------------------------------------- |
+| contentrepo_token | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx  |
 
 ## Setup
 
 Here we do any setup and fetching of values before we start the flow.
 
 ```stack
+card Checkpoint, then: FetchError do
+  update_contact(checkpoint: "basic_pregnancy_profile")
+end
```

Comment:
It seems like we're setting the checkpoints, but we're not jumping to a point in this stack if there is a checkpoint set. Just want to make sure we're aware of that (maybe it's intentional, or maybe a TODO?)

Reply:
Yeah, just setting the checkpoints right now, not doing the jump logic for it yet.